public/js/helpers/get_startTime.js

```diff
@@ -1,25 +1,26 @@
 /* global axios Cal addDateEventListeners */
+import axios from 'axios';
+import Cal from '../calendar';
+import addDateEventListeners from '../main';
 
 (function () {
-  axios
-    .get('https://nazareth-open-tourism-platform.herokuapp.com/api/v1/events')
-    .then(function (res) {
-      var data = res.data.map(event => {
-        return event.startTime;
-      });
-      // create calendar
-      var calendar = new Cal('calendar');
-      calendar.render(data);
-      addDateEventListeners();
-
-      // calendar next and prev month buttons
-      document.getElementById('next-button').onclick = function () {
-        calendar.nextMonth(data);
-        addDateEventListeners();
-      };
-      document.getElementById('prev-button').onclick = function () {
-        calendar.previousMonth(data);
-        addDateEventListeners();
-      };
+  axios.get(`${process.env.URI}/events`).then(function (res) {
```

Comment:
Are you going to convert to es6 now you're webpacking?

Reply:
Probably a different PR

Reply:
yup this PR is just one part of the main issue #106